Counter/project.json

```diff
@@ -16,12 +16,23 @@
         "Microsoft.ServiceFabric": "5.0.135",
         "Microsoft.ServiceFabric.AspNetCore.Hosting": "1.0.0-*",
         "Microsoft.ServiceFabric.Services": "2.0.135"
+      },
+      "frameworkAssemblies": {
+        "System.Runtime": { "type": "build" },
+        "System.ComponentModel": { "type": "build" }
       }
     }
   },
   "runtimes": {
     "win7-x64": { }
   },
+
+  "runtimeOptions": {
+    "configProperties": {
+      "System.GC.Server": true
+    }
+  },
+
```

Comment:
System.GC.Server is true by default per the spec: https://github.com/dotnet/cli/blob/rel/1.0.0/Documentation/specs/runtime-configuration-file.md

Is it required to explicitly specify in the project.json?

There seems another bug related also: https://github.com/dotnet/cli/issues/2485